app/models/mailing_list/signup.rb

```diff
@@ -0,0 +1,141 @@
+require "attribute_filter"
+
+module MailingList
+  class Signup
+    include ActiveModel::Model
+    include ActiveModel::Validations::Callbacks
+    include ::Wizard::ApiClientSupport
```

Comment:
Can be one for later, but I think we should rename this to something more generic like `Form::ApiClientSupport`